index.js

```diff
@@ -98,25 +131,30 @@ function myMap(array, callback) {
 }
 
 // ***** Callbacks - Question 1 *****
-
+  function triple(num) {
+    return (num * 3)
+  }
 // *** Uncomment the lines below to test
-// console.log("%cCallbacks - Question 1", "color: red")
+console.log("%cCallbacks - Question 1", "color: red")
 
-// console.log(myMap([1,2,3,4,5], triple)) 
+console.log(myMap([1,2,3,4,5], triple)) 
 // // => [3,6,9,12,15]
 
-// console.log(myMap([2,4,6,8], triple))   
+console.log(myMap([2,4,6,8], triple))   
```

Comment:
not sure if myMap was defined and then deleted right before the pull request, currently there is no callback function "myMap"